tests/settings/modify_setting_manage_dhcp_test.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import shutil
+import yaml
+from cobbler.utils import get_shared_secret
+from cobbler.remote import CobblerXMLRPCInterface
+from cobbler.api import CobblerAPI
+
+@pytest.fixture(scope="function")
+def cobbler_api() -> CobblerAPI:
+    test_yaml = '/etc/cobbler/settings_test.yaml'
+    shutil.copyfile('/etc/cobbler/settings.yaml',test_yaml)
+    CobblerAPI.__shared_state = {}
+    CobblerAPI.__has_loaded = False
+    with open(test_yaml) as settings_file:
+        new_settings = yaml.safe_load(settings_file)
+        new_settings['allow_dynamic_settings'] = True
+    with open(test_yaml, 'w+') as new_settings_file:
+        new_settings_file.write(yaml.dump(new_settings))
+    return CobblerAPI(settingsfile_location=test_yaml)
+
+
+@pytest.fixture(scope="function")
+def remote(cobbler_xmlrpc_base) -> CobblerXMLRPCInterface:
+    """
+
+    :param cobbler_xmlrpc_base:
+    :return:
+    """
+    return cobbler_xmlrpc_base[0]
+
+
+@pytest.fixture(scope="function")
+def token(cobbler_xmlrpc_base) -> str:
+    """
+
+    :param cobbler_xmlrpc_base:
+    :return:
+    """
+    return cobbler_xmlrpc_base[1]
+
+@pytest.fixture(scope="function")
+def cobbler_xmlrpc_base(cobbler_api):
+    """
+    Initialises the api object and makes it available to the test.
+    """
+    # create XML-RPC client and connect to server
+    remote = CobblerXMLRPCInterface(cobbler_api)
+    shared_secret = get_shared_secret()
+    token = remote.login("", shared_secret)
+    if not token:
+        sys.exit(1)
+    return remote, token
+
+
+class TestMiscellaneous:
+    def test_modify_setting_manage_dhcp(self, remote, token):
+        """
+        Test configuration manage_dhcp does not work
+        """
+        # Assert
+        result = remote.modify_setting("manage_dhcp",'true' , token)
+        assert result == 0
```

Comment:
A test class is not needed as there is no benefit for setup and teardown. Thus let's not use this superfluous wrapper.

```suggestion
def test_modify_setting_manage_dhcp(remote, token):
    """
    Test configuration manage_dhcp does not work
    """
    # Assert
    result = remote.modify_setting("manage_dhcp",'true' , token)
    assert result == 0
```